docs/data/base/components/number-input/UnstyledNumberInputBasic.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import NumberInput, {
+  NumberInputProps,
+  numberInputClasses,
+} from '@mui/base/Unstable_NumberInput';
+import { styled } from '@mui/system';
+
+const CustomNumberInput = React.forwardRef(function CustomNumberInput(
+  props: NumberInputProps,
+  ref: React.ForwardedRef<HTMLDivElement>,
+) {
+  return (
+    <NumberInput
+      slots={{
+        root: StyledInputRoot,
+        input: StyledInputElement,
+        incrementButton: StyledButton,
+        decrementButton: StyledButton,
+      }}
+      slotProps={{
+        incrementButton: {
+          children: '▴',
+        },
+        decrementButton: {
+          // it's flipped with CSS, the downward pointing
+          // triangle looks weird
+          children: '▴',
```

Comment:
```suggestion
          // it's flipped with CSS for a better UI
          children: '▴',
```
x the other places where it's used.